snuba/web/views.py

```diff
@@ -262,11 +281,13 @@ def health() -> Response:
     return Response(json.dumps(body), status, {"Content-Type": "application/json"})
 
 
-def parse_request_body(http_request):
+def parse_request_body(http_request: Request) -> MutableMapping[str, Any]:
     with sentry_sdk.start_span(description="parse_request_body", op="parse"):
         metrics.timing("http_request_body_length", len(http_request.data))
         try:
-            return json.loads(http_request.data)
+            body = json.loads(http_request.data)
+            assert isinstance(body, dict)
```

Comment:
What's the chances of us hitting this assertion (valid json that isn't a dict)?

Reply:
There is code later that depends on this being a dictionary